resource/templates/extract-recipe-template.html

```diff
@@ -48,18 +48,18 @@ <h2 class="settings-section-title">GPU</h2>
                <label class="control-label">GPU Memory</label>
                <div class="controls">
                   <select dku-bs-select
-                     name="gpu_memory"
-                     ng-model="config.gpu_memory"
-                     ng-options="gm.value as gm.label for gm in gpuMemory">
+                     name="gpu_memory_allocation_mode"
+                     ng-model="config.gpu_memory_allocation_mode"
+                     ng-options="gm.value as gm.label for gm in gpuMemoryAllocationMode">
                   </select>
                </div>
             </div>
-            <div ng-if="config.gpu_memory == 'custom'">
+            <div ng-if="config.gpu_memory_allocation_mode == 'memory_limit'">
                <div class="control-group">
-                  <label class="control-label">Memory limit per GPU</label>
+                  <label class="control-label">Memory allocation</label>
```

Comment:
Suggestion: for the user to better understand that this parameter is linked to the "GPU Memory" settings, you could use the same arrow symbol we used here: https://github.com/dataiku/dss-plugin-timeseries-forecast/blob/04f4dd661370695aac4e97f22f938abf713ab65c/custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json#L466
```suggestion
                  <label class="control-label"> ↳ Memory allocation</label>
```

Reply:
Also, not sure the label "Memory allocation" is easier to understand for the user than the previous term "Memory limit per GPU". In particular, the select options above refers to a "custom memory limit".

Reply:
True and true. Fixed in 419bc6e